### DISCUSSION TITLE
Build Failed  Command "yarn install" exited with 1 - NO DEPLOYMENT LOGS ON VERCEL

### DISCUSSION THREAD

**Discussion by growandwin** [vercel/community#3773] [Help]
### Page to Investigate

https://vercel.com/growandwin/howicured/GynBb7BNsTe95jzUZY55HeKZRrdd

### Steps to Reproduce

Hello, 

I was forced to update node.js when pushing a new commit to my repo. I updated node.js to v18 in project settings. Despite that fact, the build fails, and there are no logs visible on my vercel dashboard. The "Building" tab in deployment view does not want to expand nor the is no code in html. I tried in Firefox and clean Google Chrome - the same no log view is on my dashboard in vercel. 

Seems like a vercel bug to me - on my local machine everything builds fine and I haven't changed any env variables. 

Steps to reproduce:
Vercel support: please try to redeploy my latest commit qsqmi9i2x and see whether you have any logs visible (why it fails with "Build Failed  Command "yarn install" exited with 1")

**Comment by swarnava** [comment]
Hi,

Are you using Bitdefender or any other antivirus? I can see that logs are visible. Can you share a screenshot?
